examples/cosmos/ics20/denomTrace.qnt

```diff
@@ -88,20 +86,27 @@ module denomTraceTest {
     path: [],
   }
 
-  /// Finds the correct channels for a transfer according to the chain setup
-  /// described above
-  pure def getChannelsForTransfer(sourceChain: str, destChain: str): { sourceChannel: str, destChannel: str } =
-    pure val channels = Map(
-      "A" -> "channelToA",
+  /// Map from chain identifiers to their stored channels, which are maps from
```

Comment:
Not sure what "stored" means in this context, but I think the possessive pronoun is enough?

```suggestion
  /// Map from chain identifiers to their IBC channels, which are maps from
```